packages/generator/templates/app/README.md

```diff
@@ -1,8 +1,10 @@
+TODO
+
 [![Blitz.js](https://raw.githubusercontent.com/blitz-js/art/master/github-cover-photo.png)](https://blitzjs.com)
 
 This is a [Blitz.js](https://github.com/blitz-js/blitz) app.
 
-# **__name__**
+# ****name****
```

Comment:
Looks like we need to add a prettier ignore rule for this file, probably it converts `__` to `**` but we need `__` for the template engine